src/components/CreateTest.js

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import Dropdown from "./form-components/Dropdown";
+import RadioButtons from './form-components/RadioButtons'
+import apiClient from '../services/ApiClient';
+import { useState } from 'react';
+import { Link } from "react-router-dom";
+
+const CreateTest = () => {
+
+  //locations, source, comparison are hardcoded for now, ideally 
+  // we should get them from DB 
+  const locations = [
+    { label: 'us-east-2', value: 'us-east-2' },
+    { label: 'us-east-1', value: 'us-east-1' },
+    { label: 'us-west-1', value: 'us-west-1' },
+  ];
+
+  const source = [
+    { label: 'Status code', value: 'Status code' },
+    { label: 'JSON body', value: 'JSON body' },
+    { label: 'Headers', value: 'Headers' },
+    { label: 'Text body', value: 'Text body' },
+    { label: 'Response Time', value: 'Response Time'},
+  ];
+
+  const comparison = [
+    { label: 'Equals', value: 'Equals' },
+    { label: 'Not equals', value: 'Not equals' },
+    { label: 'Has key', value: 'Has key'},
+  ];
+
+  const [locationValue, setLocationValue] = useState('us-east-1');
+  const [sourceValue, setSourceValue] = useState('JSON body');
+  const [comparisonValue, setComparisonValue] = useState('Equals');
+
+  const [title, setTitle] = useState("");
+  const [frequency, setFrequency] = useState("1");
+  const [method, setMethod] = useState("");
+  const [url, setUrl] = useState("");
+  const [body, setBody] = useState("");
+  const [property, setProperty] = useState("");
+  const [target, setTarget] = useState("");
+
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const testData = {
+      "test": {
+        "title": title,
+        "locations": [locationValue],
+        "minutesBetweenRuns": Number(frequency),
+        "type": "API",
+        "httpRequest": {
+          "http_method": method,
```

Comment:
I think TestRunner is now expecting just `method`

Reply:
perfect. Will change that.